src/Microsoft.PowerShell.Commands.Management/commands/management/ContentCommandBase.cs

```diff
@@ -619,7 +619,7 @@ internal virtual bool CallShouldProcess(string path)
                             notSupported.ErrorRecord,
                             notSupported));
                 }
-                catch (DriveNotFoundException driveNotFound)
+                catch (System.Management.Automation.DriveNotFoundException driveNotFound)
```

Comment:
I see a lot of conflicts with  System.IO.DriveNotFoundException.

Can we rename System.Management.Automation.DriveNotFoundException to PSDriveNotFoundException?

Reply:
That would be a breaking change. I do not believe it is needed right now.